uptrends/client.go

```diff
@@ -0,0 +1,68 @@
+package uptrends
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
```

Comment:
Highly suggest taking a look at https://github.com/go-resty/resty

Might provide a better base to create an API Client off of.

Reply:
Cool! Thanks for the recommendation @dlamotte .